src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateSummary.tsx

```diff
@@ -10,39 +9,26 @@ import {DEFAULT_DAY_END_TIME, DEFAULT_DAY_START_TIME} from 'app/utils/dates';
 import {t} from 'app/locale';
 import space from 'app/styles/space';
 
-class DateSummary extends React.Component {
-  static propTypes = {
-    /**
-     * Start date value for absolute date selector
-     * Accepts a JS Date or a moment object
-     *
-     * React does not support `instanceOf` with null values
-     */
-    start: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+type Props = {
+  start: Date;
+  end: Date;
```

Comment:
This most likely should be `type RelaxedDateType = string | number | Date;` like timeSince. The proptypes also allowed strings

Reply:
We have a type `DateString` that could work here, it doesn't include `number` currently. It is used in a bunch of places that interact with moment, and query string values. moment also has `moment.MomentInput` which could work here.

Reply:
ooh gotcha yea I'll opt for DateString then

Reply:
ooh gotcha yea I'll just use moment.MomentInput then